src/components/Coin.jsx

```diff
@@ -22,7 +22,7 @@ const Coin = (props) => {
         </div>
         <div className="container-column">
             <div className="title">{name}</div>
-            <div className="title">{symbol}</div>
+            <div className="coin-symbol">{symbol}</div>
           </div>
       </div>
```

Comment:
```jsx
  const handlerDetails = (id) => {
    window.location.href = `details/${id}`;
  };
```
- [ ] Kindly make sure your page is not reloading when I want to show the details of an item as the requirements emphasize showing the details page without reloading the page. you can do it by using the `useNavigate`. check this example [on how to use navigate pages without reloading](https://www.geeksforgeeks.org/reactjs-usenavigate-hook/#:~:text=The%20useNavigate()%20hook%20is,URL%2C%20forward%20or%20backward%20pages.).

Example in your project :point_down: 

```jsx
import React from 'react';
import PropTypes from 'prop-types';
import { useNavigate } from 'react-router-dom';

const Coin = (props) => {
  const navigate = useNavigate();

  const {
    id,
    name,
    image,
    symbol,
  } = props;

  const handlerDetails = (id) => {
    navigate(`/details/${id}`);
  };

  return (
    <>
      <div className="container-column">
        <div className="coin-container">
          <img className="coin-img" src={image} alt={name} />
          <button type="button" className="coin-button" onClick={() => handlerDetails(id)}>-></button>
        </div>
        <div className="container-column">
          <div className="title">{name}</div>
          <div className="coin-symbol">{symbol}</div>
        </div>
      </div>
    </>
  );
};

Reply:
done